### ISSUE TITLE
Lazyload Youtube iframes: failing accessibility audits

### ISSUE THREAD

**Issue by michaelbourne** [wp-media/rocket-lazyload-common#98]
I had intended to make the Issue here as it is the common repo, but a member of your support staff made one in the core plugin instead.

I'd like to reference it here since it applies directly to this library.

https://github.com/wp-media/wp-rocket/issues/4436
